share/jupyterhub/templates/page.html

```diff
@@ -51,15 +50,15 @@ <h1 class="modal-title" id="{{key}}-label">{{title}}</h1>
           paths: {
             components: '../components',
             jquery: '../components/jquery/dist/jquery.min',
-            bootstrap: '../components/bootstrap/dist/js/bootstrap.min',
+            // bootstrap: '../components/bootstrap/dist/js/bootstrap.bundle.min',
             moment: "../components/moment/moment",
           },
-          shim: {
-            bootstrap: {
-              deps: ["jquery"],
-              exports: "bootstrap"
-            },
-          }
+          // shim: {
+          //   bootstrap: {
+          //     deps: ["jquery"],
+          //     exports: "bootstrap"
+          //   },
+          // }
```

Comment:
Something to cleanup or add additional comment about in order to clarify why its commented out